ultravox/tools/ds_tool.py

```diff
@@ -132,6 +138,7 @@ def main(args: DatasetToolArgs):
         "token": token,
         "revision": args.upload_branch,
         "private": args.private,
+        "split": args.upload_split,
```

Comment:
Just noting that this is going to conflict with my PR, but it's okay to merge yours first.